L03-generics/src/main/java/ru/otus/homework/CustomerReverseOrder.java

```diff
@@ -0,0 +1,19 @@
+package ru.otus.homework;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class CustomerReverseOrder {
+
+    private final List<Customer> customers = new LinkedList<>();
+
+    public void add(Customer customer) {
+        customers.addFirst(customer);
+    }
+
+    public Customer take() {
+        Customer customer = customers.getFirst();
+        customers.remove(customers.getFirst());
+        return customer;
```

Comment:
```suggestion
        return customers.pop();
```

Метод `pop()` делает то же самое - возвращает и удаляет значение.

Reply:
поправил